webapp/platform/client/src/client4.ts

```diff
@@ -4201,6 +4201,28 @@ export default class Client4 {
             {method: 'delete', body: JSON.stringify(deletionRequest)},
         );
     }
+
+    getGroupMessageMembersCommonTeams = (channelId: string) => {
+        const u = `${this.getChannelRoute(channelId)}/common_teams`;
+        console.log(u);
```

Comment:
Unneeded console.log? (shouldn't this be found by lint?)

Reply:
Done.